### DISCUSSION TITLE
Any audio quality difference when using with Spotify Premium?

### DISCUSSION THREAD

**Discussion by githupper** [KRTirtho/spotube#457] [Q&A]
Hi,

I checked both Active and Closed issues and did not any information on the above. 

Would using it with Spotify Premium gets me 320Kbps or just whatever "High" quality gives me. 

If so, what is the bitrate of "High"? or the benefit of using Spotube with Spotify Premium?

**Comment by RaptaG** [comment]
Using a Spotify account on Spotube with a Premium subscription will do nothing more than a non-Premium account, songs are streamed from YouTube. Lower sound quality is there for resource saving purposes.
